docs/application-security/intro.md

```diff
@@ -0,0 +1,37 @@
+# Application Security
+
+## Storing and Retrieving Application Secrets
+
+### Environment: Azure
+
+When using Azure and Github actions, care should taken to not run github actions in debug mode as the `azure/webapps-deploy@v2` function required to deploy the app to azure will print out all the values saved in the App service configuration settings as the result of an internal method called `getAppSettings`.
+
+There is a risk if any of the App service configuration values are sensitive because, they would be printed into the github actions log if the workflow is run in debug mode.
```

Comment:
```suggestion
There is a risk if any of the App service configuration values are sensitive because they would be printed into the github actions log if the workflow is run in debug mode.
```